docs/sources/tempo/configuration/network/tls.md

```diff
@@ -96,3 +96,159 @@ The above structure can be set on the following receiver configurations:
 - `distributor.receivers.zipkin.tls`
 - `distributor.receivers.jaeger.protocols.grpc.tls`
 - `distributor.receivers.jaeger.protocols.thrift_http.tls`
+
+### Configure TLS with Helm
+
+To configure TLS with the Helm chart, you must have a TLS key-pair and CA certificate stored in a Kubernetes secret.
```

Comment:
Will this example values.yaml file work with any Tempo or GET Helm chart? Or is it specifically designed to use tempo-distributed? 

Asking because I thought about adding a link to the tempo-distributed Helm chart and doc here, but then I wasn't sure if this is specifically part of that. 

Either way, I'm linking from the tempo-distributed doc to here for TLS.

Reply:
This is only for the `tempo-distributed` helm chart.